css/src/elementor.css

```diff
@@ -12,7 +12,9 @@
 }
 
 .yoast h2 {
-	color: var(--yoast-color-font-default);
+	color: var(--yoast-color-dark);
```

Comment:
In the Gutenberg sidebar the color for h2's are `#1e1e1e`, this var is `#303030`.

I understand why you did it like this, but for next time it's nice to document your rationale in the PR text.